src/MToken.sol

```diff
@@ -78,6 +79,17 @@ contract MToken is IMToken, ContinuousIndexing, ERC20Extended {
 
     /// @inheritdoc IMToken
     function burn(address account_, uint256 amount_) external onlyPortal {
+        uint256 spenderAllowance_ = allowance[account_][msg.sender];
```

Comment:
```suggestion
        uint256 portalAllowance_ = allowance[account_][portal];
```

Reply:
Fixed in this commit: https://github.com/m0-foundation/protocol/pull/190/commits/ed31631c7f2fc0ed50176cd0f46df6c91d9aa9d2